src/jshint.js

```diff
@@ -5011,11 +4804,7 @@ var JSHINT = (function() {
     }
   }
 
-  var warnUnused = function(name, tkn, type, unused_opt) {
-    var line = tkn.line;
-    var chr  = tkn.from;
-    var raw_name = tkn.raw_text || name;
-
+  var getUnusedOption = function(unused_opt) {
```

Comment:
yep - both `getUnusedOption` and `warnUnused` are now only used within scope manager, so they can both be moved in there.